Libraries/Utilities/Platform.android.js

```diff
@@ -20,19 +19,17 @@ export type PlatformSelectSpec<A, D> = {
 const Platform = {
   OS: 'android',
   get Version() {
-    const constants = NativeModules.PlatformConstants;
-    return constants && constants.Version;
+    const { Version } = NativePlatformConstants.getConstants();
+    return Version || 0;
```

Comment:
I wasn't sure if there was a better was of forcing the value going from here to be non nullable